### PR TITLE
Route cloudbackup status and statechange commands through sdk for auth.

### PR DESCRIPTION
**Pull request by talakad**
Signed-off-by: veda <veda@portworx.com>

**What this PR does / why we need it**:
Routes cloud backup Status and StaeChange commands through sdk-atuh.
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: api/server/sdk/cloud_backup.go [hunk 1]
```diff
@@ -354,10 +354,12 @@ func (s *CloudBackupServer) StateChange(
 
 	var rs string
 	switch req.GetRequestedState() {
-	case api.SdkCloudBackupRequestedState_SdkCloudBackupRequestedStatePause:
-		rs = api.CloudBackupRequestedStatePause
-	case api.SdkCloudBackupRequestedState_SdkCloudBackupRequestedStateResume:
-		rs = api.CloudBackupRequestedStateResume
+	/*
```
Comment (lpabon) [inline api/server/sdk/cloud_backup.go]: Can you add comments to why those lines are commented out? That way the next person reading the code will know why.

File: api/server/backup.go [hunk 2]
```diff
@@ -302,13 +332,26 @@ func (vd *volAPI) cloudBackupStateChange(w http.ResponseWriter, r *http.Request)
...
+
+	// Get gRPC connection
+	conn, err := vd.getConn()
+	if err != nil {
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	err = d.CloudBackupStateChange(stateChangeReq)
+	volumes := api.NewOpenStorageCloudBackupClient(conn)
```
Comment (lpabon) [inline api/server/backup.go]: nit: this is not the volumes api, it's the cloudbackups api.

File: api/server/backup.go [hunk 3]
```diff
@@ -230,22 +230,52 @@ func (vd *volAPI) cloudBackupStatus(w http.ResponseWriter, r *http.Request) {
...
-		notFound(w, r)
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	backupStatusResp, err := d.CloudBackupStatus(&backupStatus.CloudBackupStatusRequest)
+	// Get gRPC connection
+	conn, err := vd.getConn()
 	if err != nil {
-		if err == volume.ErrInvalidName {
```
[the remark this answers is not in the document]
Comment (talakad) [reply api/server/backup.go]: ok. Done.

File: api/server/sdk/cloud_backup.go (hunk 1 above)
Comment (talakad) [reply api/server/sdk/cloud_backup.go]: Done.

File: api/server/backup.go (hunk 2 above)
Comment (talakad) [reply api/server/backup.go]: Done.

**Pull request merged**